app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -113,6 +114,8 @@ const Summary = ({ experiment, refetch }: SummaryProps) => {
 
       <hr />
 
+      <PreviewURL {...experiment} />
```

Comment:
If I understand the issue correct, this Preview URL is only valid for Live or Preview experiments, yeah? If that's the case we should conditionally render this so it doesn't show up for states like Draft, Complete, etc.